react/components/CheckCustomerSchema.tsx

```diff
@@ -23,23 +23,27 @@ const CheckCustomerSchema = ({ isAdmin }: { isAdmin: boolean }) => {
   return (
     <Fragment>
       {isInvalidSchema && (
-        <Alert
-          type="error"
-          action={{
-            label: formatMessage(
-              isAdmin ? adminMessages.checkItOut : storeMessages.checkItOut
-            ),
-            onClick: () =>
-              window.open(
-                'https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite',
-                '_blank'
+        <div className="mb5">
```

Comment:
This is the only real change in this file, moving that extra spacing I mentioned from `OrganizationTable` to here. The rest is just indentation.